File: C#/03-Operadores Aritméticos/Precedencia.cs
using System;

class Precedencia 
{
    static void Main(string[] args)
    {
//A ordem de precedencia dos operadores em C# é:
//1-Expressões entre parenteses()
//2-Multiplicação e Divisão
//3-Soma e Subtração 

//Vamos calcular uma media de 2 valores
        int x = 9;
        int y = 10;
//Sem o uso dos parenteses primeiro seria resolvida a divisão y/2 invalidando a média        
        double media = (x + y) / 2d; //Queremos o resultado do tipo double(mesmo que seja divisão sem resto)
        Console.WriteLine("A média entre " +x +" e " +y +" é " +media);
        Console.ReadLine();
    }
}

File: Java/03-Operadores Aritméticos/Precedencia.java
class Precedencia {
    public static void main(String[] args) {
        String nome = "Pamela";
        int val1 = 8;
        int val2 = 9;
        int val3 = 9;

        /*
         * A pecedencia dos operadores aritmeticos é Parenteses (opcional) Multiplicação
         * e Divisão Soma e Subtração Operadores de mesma precedencia é feito da
         * esquerda para a direita
         */
        double media = (val1 + val2 + val3) / 3;

        System.out.println("A media final de " + nome + " foi " + media);
    }
}